CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ Changes
 * [Developer]: Updated to node `v12.8.1`.
 * [Developer]: Updated `eslint` to fix security vulnerability.
 * [Admin]: Added NCBI Upload FTP option to disable/enable passive mode for FTP `ncbi.upload.ftp.passive`.
+* [UI/Developer]: Updated ant.design and ag-grid to fix issue with horizontal scrolling in firefox.
```

Comment:
Is development pushed up through your base branch, this should not be showing here.

Reply:
I think it still needs to be pushed

Reply:
So i have it pushed up through the base branch and it is still showing as a change. It also added the `UPGRADING.md` file below

Reply:
The problem with this is that your base branch `phac-nml:analysis/_base_results` is behind development.  I think was you did was merged development straight into this branch, or forgot to push the base branch up after merging dev into it.